### PR TITLE
test(gradual): add pipeline for gradual test with tablets

### PR DESCRIPTION
**Pull request by juliayakovlev**
Add Jenkins pipeline for predefined throughput steps test with tablets

### Testing

- [ ]

### PR pre-checks (self review)

- [ ] I added the relevant `backport` labels
- [ ] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: jenkins-pipelines/performance/branch-perf-v16/scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-predefined-throughput-steps-vnodes-tablets.jenkinsfile
Comment (roydahan) [inline jenkins-pipelines/performance/branch-perf-v16/scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-predefined-throughput-steps-vnodes-tablets.jenkinsfile:1]: the name shouldn't include vnodes

**Review by roydahan — CHANGES_REQUESTED** [review]

File: jenkins-pipelines/performance/branch-perf-v16/scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-predefined-throughput-steps-vnodes-tablets.jenkinsfile
Comment (juliayakovlev) [reply jenkins-pipelines/performance/branch-perf-v16/scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-predefined-throughput-steps-vnodes-tablets.jenkinsfile:1]: changed

**Review by fruch — DISMISSED** [review]
LGTM

**Comment by fruch** [comment]
> I would recommend adding it to the triggers as well
> 
> @roydahan you think this should be triggered weekly ? or same as other tablets cases, every 3 weeks ?

@roydahan WDYT ?

**Comment by roydahan** [comment]
Weekly.

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**